components/permission-controller/controllers/namespace_controller.go

```diff
@@ -0,0 +1,148 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1beta1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	RolebindingName = "namespace-admin"
+	RoleRefKind     = "ClusterRole"
+	RoleRefName     = "kyma-admin"
+
+	subjectKindGroup  = "Group"
+	subjectKindUser   = "User"
+	SubjectStaticUser = "admin@kyma.cx"
```

Comment:
Please change that to `namespace.admin@kyma.cx`
```suggestion
	SubjectStaticUser = "namespace.admin@kyma.cx"
```
`admin@kyma.cx` already has `kyma-admin` role bound cluster-wide

Reply:
see "address review comments 3"